playbooks/user/stageuser_activate.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Tests
+  hosts: ipaserver
+  become: true
+  gather_facts: false
+
+  tasks:
+  - name: User pinky activate 
+    ipastageuser:
+      ipaadmin_password: 'Azsx$123Azsx$123'
+      name: pinky
+      state: activate
```

Comment:
I'm not a fan of "activate" here, as "activate" is an action, not a state. Not sure if "active" is good.

Reply:
@rjeffman Understood.  Active would be a better choice for state.  There will be an action to move the user from stage users to active users. I'll look into this further and provide a suggestion.  Thanks again for the response.

Reply:
`activated` would be better, as we have have `enabled`, `disabled` and similar in other modules.